packages/fast-components-class-name-contracts-base/src/dialog/index.ts

```diff
@@ -7,6 +7,11 @@ export interface DialogClassNameContract {
      */
     dialog?: string;
 
+    /**
+     * The wrapper of the dialog contents
+     */
+    dialog_wrapper?: string;
```

Comment:
should we be more specific to what is being wrapped? for example "dialog_contentRegionWrapper"